Whiskey/Whiskey.psd1

```diff
@@ -1,156 +1,157 @@
+                            'Get-WhiskeyCredential',
+                            'Install-WhiskeyNodeJs',
+                            'Install-WhiskeyTool',
+                            'Invoke-WhiskeyNodeTask',
+                            'Invoke-WhiskeyNUnit2Task',
+                            'Invoke-WhiskeyPester3Task',
+                            'Invoke-WhiskeyPester4Task',
+                            'Invoke-WhiskeyPipeline',
+                            'Invoke-WhiskeyBuild',
+                            'Invoke-WhiskeyTask',
+                            'New-WhiskeyContext',
+                            'Publish-WhiskeyBuildMasterPackage',
+                            'Publish-WhiskeyNuGetPackage',
+                            'Publish-WhiskeyProGetUniversalPackage',
+                            'Publish-WhiskeyBBServerTag',
+                            'Register-WhiskeyEvent',
+                            'Resolve-WhiskeyNuGetPackageVersion',
+                            'Resolve-WhiskeyPowerShellModuleVersion',
+                            'Resolve-WhiskeyTaskPath',
+                            'Resolve-WhiskeyVariable',
+                            'Set-WhiskeyBuildStatus',
+                            'Stop-WhiskeyTask',
+                            'Uninstall-WhiskeyTool',
+                            'Unregister-WhiskeyEvent'
+                         );
+
+    # Cmdlets to export from this module
+    CmdletsToExport = @( )
+
+    # Variables to export from this module
+    #VariablesToExport = '*'
+
+    # Aliases to export from this module
+    AliasesToExport = '*'
+
+    # DSC resources to export from this module
+    # DscResourcesToExport = @()
+
+    # List of all modules packaged with this module
+    # ModuleList = @()
+
+    # List of all files packaged with this module
+    # FileList = @()
+
+    # Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
+    PrivateData = @{
+
+        PSData = @{
+
+            # Tags applied to this module. These help with module discovery in online galleries.
+            Tags = @( 'build', 'pipeline', 'devops', 'ci', 'cd', 'continuous-integration', 'continuous-delivery', 'continuous-deploy' )
+
+            # A URL to the license for this module.
+            LicenseUri = 'https://www.apache.org/licenses/LICENSE-2.0'
+
+            # A URL to the main website for this project.
+            ProjectUri = 'https://github.com/webmd-health-services/Whiskey'
+
+            # A URL to an icon representing this module.
+            # IconUri = ''
+
+            # ReleaseNotes of this module
+            ReleaseNotes = @'
+* You can now specify a custom version of NUnit 2. Set the `Version` property on the `NUnit2` task.
+* Added support for `PackageName` and `DeployTo` properties to `Publish-WhiskeyBuildMasterPackage` task function
```

Comment:
Can you be a little more descriptive about the change you made? Maybe move it out of a bullet point and into its own paragraph?

Reply:
Yup. Updated.